src/components/SQFormDialogStepper/SQFormDialogStepper.tsx

```diff
@@ -192,13 +246,13 @@ export function SQFormDialogStepper({
             <Divider />
             {steps.length > 1 && (
               <Grid container classes={stepperClasses}>
-                <Stepper activeStep={activeStep} classes={classes}>
+                <Stepper
+                  activeStep={activeStep}
+                  classes={classes}
+                  alternativeLabel
```

Comment:
nit: can we do `alternativeLabel={true}` here. It's the preferred way to pass a boolean as prop